docs/README.md

```diff
@@ -42,7 +42,7 @@ npm install --global yarn
 yarn
 ```
 
-> Para mais detalhes ou informações sobre a configuração em outros sistemas operacionais veja em ["como configurar o ambiente de desenvolvimento"](/docs/CONTRIBUTING.md#como-configurar-o-ambiente-de-desenvolvimento).
+> Para mais detalhes ou informações sobre a configuração em outros sistemas operacionais veja em ["como configurar o ambiente de desenvolvimento"](https://github.com/okfn-brasil/querido-diario-frontend/blob/main/docs/CONTRIBUTING.md#como-configurar-o-ambiente-de-desenvolvimento).
```

Comment:
O problema na verdade é apenas a barra no começo, o GitHub sabe resolver o caminho sem ela.

```suggestion
> Para mais detalhes ou informações sobre a configuração em outros sistemas operacionais veja em ["como configurar o ambiente de desenvolvimento"](docs/CONTRIBUTING.md#como-configurar-o-ambiente-de-desenvolvimento).
```

Reply:
> O problema na verdade é apenas a barra no começo, o GitHub sabe resolver o caminho sem ela.

@lguima Criei uma nova branch para testar sua sugestão e está dando erro 404.
Segue o link: https://github.com/AlexJBSilva/querido-diario-frontend/tree/patch-1_NOK_sem_Barra

Reply:
Ahh, é por que o `README.md` tá dentro da pasta `docs/`, então teria que remover, na verdade.

```suggestion
> Para mais detalhes ou informações sobre a configuração em outros sistemas operacionais veja em ["como configurar o ambiente de desenvolvimento"](CONTRIBUTING.md#como-configurar-o-ambiente-de-desenvolvimento).
```

Reply:
Acredito que aí vai continuar dando erro ao tentar acessar o link a partir da página principal do projeto.
Mas testo essa sugestão hoje à noite. Se funcionar atualizo esse PR.

Reply:
Houston, we have a BUG!

Testei essa nova sugestão e também deu erro.

Então, busquei a [documentação oficial sobre links relativos no README.md](https://docs.github.com/pt/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes#relative-links-and-image-paths-in-readme-files) e criei outras branches para testar as possibilidades, mas sem sucesso.

Continuando a pesquisa, achei [essa issue](https://github.com/orgs/community/discussions/67750) recente, relatando o mesmo problema.

Como [acusti informa](https://github.com/orgs/community/discussions/70664#discussioncomment-7348156):
**Resumindo:** links relativos contidos em um bloco de marcação 'não parágrafo' (linhas iniciadas com `-` ou `>` ) ficam quebrados no arquivo leia-me do repositório, quando renderizado na página inicial do repositório.

**Possíveis soluções de contorno enquanto o Github não resolve o bug:**
1. Mova o link para fora de qualquer bloco de marcação que não seja de parágrafo.
Por exemplo:  remova o `>` ou `-`  da frente do texto que contém o link quebrado.

3. Atualize o link para ter uma barra inicial e incluir o prefixo `/user/repo-name/blob/main/` do repositório.
Por exemplo: `/okfn-brasil/querido-diario-frontend/blob/main/docs/CONTRIBUTING.md#como-configurar-o-ambiente-de-desenvolvimento`.

Reply:
Eita, boa! Eu iria com a primeira alternativa de remover do quote block, não vejo que vá mudar em nada na documentação, o que acha?

Reply:
Primeira alternativa (remover o `>` da frente do texto) aplicada.
Testes OK.
PR atualizado.